Controller/RacetimeChannel.cs

```diff
@@ -342,41 +342,20 @@ private void UpdateRaceData(RaceMessage msg)
                     //the race is starting
                     if ((r == RaceState.Open || r == RaceState.OpenInviteOnly) && nr == RaceState.Starting)
                     {
-                        m.CurrentState.Run.Offset = DateTime.UtcNow - msg.Race.StartedAt;
                         m.Reset();
```

Comment:
Can we get rid of this `Reset()` call as well? We shouldn't reset the timer if the runner has already started it manually.

Reply:
👍 0308c41